ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/sync/state/SyncState.java

```diff
@@ -49,17 +51,22 @@ public SyncState(final Blockchain blockchain, final EthPeers ethPeers) {
         });
   }
 
-  private void publishSyncStatus() {
+  @VisibleForTesting
```

Comment:
(discussion) : why `@VisibleForTesting` if public ?

Reply:
I didn't know this before but `@VisibleForTesting` is for documentation purposes only! I used to think it actually changed the visibility for the tests in some special way. It's to communicate that if not for the tests, this method would be less visible.